examples/secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner/3-Task-Request-Withdraw-200_OK.json

```diff
@@ -83,6 +83,21 @@
       ]
     }
   ],
+  "extension": [
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation",
+      "extension": [
+        {
+          "url": "numberOfRepeatsAllowed",
```

Comment:
Sorry for not noticing the previously, I would have expected that we would only have numberOfRepeatsIssued here, not numberOfRepeatsAllowed

Reply:
Added this change for the example and the EPSAT withdraw example